src/extensions/extensionsHelper.ts

```diff
@@ -61,6 +60,13 @@ export const EXTENSIONS_BUCKET_NAME = envOverride(
   "FIREBASE_EXTENSIONS_UPLOAD_BUCKET",
   "firebase-ext-eap-uploads"
 );
+const autopopulatedParamNames = [
```

Comment:
Should this be in CONSTANT_CASE?

https://google.github.io/styleguide/tsguide.html#identifiers

(Looks like we go back and forth, but my reading of the style guide is that they all be in CONSTANT_CASE).

Reply:
Good point, I'll convert all of these

Reply:
Ended up leaving logPrefix as is for now because its used in a dozen files, and i didn't want to scope creep quite that much